Resources/Prototypes/_Sunrise/InventoryTemplates/penguin_inventory_template.yml

```
@@ -0,0 +1,13 @@
+  id: penguin
+  slots:
+  - name: mask
+    slotTexture: mask
+    slotFlags: MASK
+    uiWindowPos: 0,2
+    strippingWindowPos: 1,1
+    displayName: Mask
+    whitelist:
+      tags:
+        - Cigarette
+        - Cigar
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)